llvm/lib/TableGen/Record.cpp

```diff
@@ -917,6 +917,13 @@ const Init *UnOpInit::Fold(const Record *CurRec, bool IsFinal) const {
       return NewInit;
     break;
 
+  case INITIALIZED:
+    if (isa_and_nonnull<UnsetInit>(LHS))
```

Comment:
This choice seems interesting - `LHS` shouldn't ever be null here, right?

Reply:
I recall  some of the surrounding code being careful about NULLs, so ... seemed worth doing

Reply:
On further inspection, yeah, this doesn't need a nullness check, fixed